flink-runtime/src/test/java/org/apache/flink/runtime/operators/drivers/DriverTestData.java

```diff
@@ -100,7 +100,9 @@ public static List<Tuple2<StringValue, IntValue>> createReduceMutableDataGrouped
 
     public static final void compareTupleArrays(Object[] expected, Object[] found) {
         if (expected.length != found.length) {
-            Assert.assertEquals("Length of result is wrong", expected.length, found.length);
+            assertThat(found.length)
+                    .withFailMessage("Length of result is wrong")
+                    .isEqualTo(expected.length);
```

Comment:
```suggestion
            assertThat(found)
                    .withFailMessage("Length of result is wrong")
                    .hasSameSizeAs(expected);
```

BTW, the `if` and the `assert` is odd. 

- If the length are not same, assert them to equal
- If the length are same, don't assert

That means we can assert directly, the if can be removed, right?